app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2>Log in</h2>
```

Comment:
- [ ] Good job so far but I think you can improve you login page by making the two inputs and submit button similar to the one on the wireframe 👇  . Please fix it in order to make your project met the requiement about sticking wireframe for login 😄 

#### Your login design
![errorslog](https://github.com/edahigure/blog-app-2/assets/6615394/a82fe356-9258-48af-b46a-2d55378b8a61)


#### Wirefame
![blog_login](https://github.com/edahigure/blog-app-2/assets/6615394/8440e369-4f6d-4c0c-a03a-ec4da78a2e34)